.travis.yml

```diff
@@ -2,11 +2,27 @@ language: c
 compiler:
   - clang
   - gcc
+
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -qq libcunit1 libcunit1-doc libcunit1-dev
+
 before_script:
   - cmake .
+
 script:
   - make
   - make test
+
+after_success:
+  - git stash --all
+  - sh scripts/update-minor.sh
+
+deploy:
+  provider: releases
+  api_key:
+    secure: $GITHUB_TOKEN
```

Comment:
doesn't make any sense this, this is the deploy and should upload a file to a release.